src/Firehose.Web/Authors/ShankarMadeshvaran

```diff
@@ -0,0 +1,40 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.ServiceModel.Syndication;
+using System.Web;
+using Firehose.Web.Infrastructure;
+
+namespace Firehose.Web.Authors
+{
+	public class ShankarMadeshvaran : IAmACommunityMember 
```

Comment:
You need to add this in the csproj file as well.

Reply:
File type needs to be `.cs`

Reply:
> csproj

where is csproj located

Reply:
how to added my author file to csproj

Reply:
Clone your fork and open the solution in VS and add the file to the authors folder. Or simply edit the csproj file manually.

Reply:
i just edited csproj file manually via github editor and created another pull request.

Reply:
Please stick to _one_ pull request.

Reply:
What should I do now? Did you edit something? or I need to add my author file to .csproj

Reply:
I think I need to change my blog Url and Feed Url. Should I delete this pull request and create a new one?